docs/using/examples/api-examples.rst

```diff
@@ -0,0 +1,14 @@
+:tocdepth: 2
+
+====================
+Flocker API Examples
+====================
+
+You can find below an example of using the :ref:`Flocker REST API<api>` with Python.
+
+Examples of using the Flocker API with languages such as `cURL`, `Go`, and `Ruby` will be available soon.
```

Comment:
Single backticks are for italics, I think that double backticks are what you want.

Is it useful to say what will be available soon? It adds risk in case the roadmap changes (and this is software, it does :smile: ).